docs/src/development.md

```diff
@@ -228,6 +228,61 @@ corresponding Vim plugin
 mode [julia-emacs](https://github.com/JuliaEditorSupport/julia-emacs).
 
 
+## Debugging
+Julia offers several options for debugging. A classical debugger is available with the 
+[Debugger.jl](https://github.com/JuliaDebug/Debugger.jl) package or in the 
+[Julia extension for VS Code](https://www.julia-vscode.org/docs/stable/userguide/debugging/). 
+However, it can be quite slow and does currently not work properly for Trixi. The 
+[Infiltrator.jl](https://github.com/JuliaDebug/Infiltrator.jl) package on the other hand does not 
+offer all features of a full debugger, but is a fast and simple tool that allows users to set 
+breakpoints to open a local REPL session and access the call stack and variables.
+
+### Infiltrator
+The Infiltrator package provides fast breakpoints using the ```@infiltrate``` command to drop the 
+user into a local REPL session, where it is possible to access local variables, see the call stack 
+and execute statements.
+
+The package can be installed in the Julia REPL by executing 
+```julia-repl
+(@v1.8) pkg> add Infiltrator
+```
+
+To load the package in the Julia REPL execute
+```julia-repl
+julia> using Infiltrator
+```
+
+Breakpoints can be set by adding a line with the ```@infiltrate``` macro at the respective position 
+in the code. Use [Revise](@ref interactive-use-of-julia) if you want to set and delete breakpoints 
+interactively. 
```

Comment:
```suggestion
Breakpoints can be set by adding a line with the ```@infiltrate``` macro at the respective position 
in the code. Use [Revise](@ref interactive-use-of-julia) if you want to set and delete breakpoints 
in your package code without having to restart Julia. 
```

Reply:
Done in e933579e5f9a007f4e406f422a5d4f0933b62e2c